### ISSUE TITLE
force attribute not working

### ISSUE THREAD

**Issue by benweizhu** [yargalot/gulp-accessibility#7]
Hi,

I expected when I set force to false, if there is any error like: 
ERROR WCAG2AAA.Principle1.Guideline1_3.1_3_1_AAA.G141

The gulp task should failed with exit -1.

But the result is Finished 'accessibility'
